kong.conf.default

```diff
@@ -270,8 +270,14 @@
                                  # for all nodes.  Under `pki` mode it
                                  # should be a different certificate for each
                                  # DP node.
+                                 #
+                                 # The certificate can be configured on this
+                                 # property with either of the following values:
+                                 # * absolute path to the certificate
+                                 # * certificate content
+                                 # * base64 encoded certificate content
```

Comment:
we discussed something similar with @bungle during implementation. We went for this approach (using each property to accept different values as described in kong.conf.default, with a fallback kind of logic) to avoid duplicating properties, which should then be considered mutually exclusive. I guess it felt more simple from a user's perspective.